template/web/src/modules/Scrolly/RichText/index.tsx

```diff
@@ -0,0 +1,81 @@
+/* eslint-disable react/display-name */
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import React, { FC } from 'react'
+import { graphql } from 'gatsby'
+import BlockContent from '@sanity/block-content-to-react'
+import Link from '@/atoms/Link'
+import Text from '@/atoms/Text'
+import Box from '@/atoms/Box'
+import { SanityButton } from '@/atoms/Button'
+
+const Block = (props: any): JSX.Element | null => {
+  const { style } = props.node
+  const variants = {
+    h1: 'head',
+    h2: 'head',
+    h3: 'subhead',
+    h4: 'subhead',
+    caption: 'caption',
+    p: 'normal',
+    span: 'normal',
+    normal: 'normal'
+  }
+  if (props.children) {
+    return (
+      <Text
+        as={style === 'normal' ? 'p' : style === 'caption' ? 'span' : style}
+        variant={variants[style]}
+      >
+        {props.children}
+      </Text>
+    )
+  } else return null
+}
+
+const serializers = {
```

Comment:
can we insert the regular rich text component here to avoid duplicate code?

Reply:
Done in commit: 705917f